### ISSUE TITLE
docker compose fails

### ISSUE THREAD

**Issue by tdcam** [ansible/awx#14993]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I might not receive a timely response.
- [X] I am **NOT** reporting a (potential) security vulnerability. (These should be emailed to `security@ansible.com` instead.)

### Bug Summary

Every time I try to use docker compose, the awx_tools_1 container build fails at error: could not create awx.egg-info. Permission denied``

### AWX version

24.0.0

### Select the relevant components

- [ ] UI
- [ ] UI (tech preview)
- [x] API
- [ ] Docs
- [ ] Collection
- [ ] CLI
- [ ] Other

### Installation method

docker development environment

### Modifications

no

### Ansible version

ansible-9.3.0-1.fc39.noarch

### Operating system

Fedora 39

### Steps to reproduce

Follow the instructions at https://github.com/ansible/awx/blob/devel/tools/docker-compose/README.md.
When it gets to the awx_tools_1 container build, it crashes 
![Screenshot_2024-03-14_00-21-49](https://github.com/ansible/awx/assets/63890286/934c5ca3-2c53-4980-8d4e-ba92db50c8a2)

### Expected results

Build all the containers and start AWX

### Actual results

error: could not create awx.egg-info. Permission denied``

**Comment by tdcam** [comment]
Sorry, mismarked it as collections. Changed it to CLI since I'm trying to install from the CLI. Let me know if this is wrong.

**Comment by fosterseth** [comment]
are you running make docker-compose with sudo? better to not run as root

for reference my awx folder has the following permissions

```
drwxr-xr-x. 1 sbf sbf     3078 Mar 14 13:21 .
drwxr-xr-x. 1 sbf root     160 Feb 19 15:28 awx.egg-info
```

**Comment by tdcam** [comment]
No, sir. I'm running it as a regular user. Now, my account is in the wheel group, but I just ran "make docker-compose"I can definitely take my user out of the wheel group and test again when I get to the hotel.
